circle.yml

```diff
@@ -4,8 +4,7 @@ machine:
   node:
     version: v5.5.0
   environment:
-    NODE_012_VERSION: v0.12.0
-    NODE_4_VERSION: v4.2.2
+    NODE_4_VERSION: v4.5.0
```

Comment:
I would be totally happy moving to the current node LTS, and removing all parallelization from the build. Or at most, do two: LTS and current.